mkt/urls.py

```diff
@@ -18,6 +18,11 @@
 from mkt.users.views import logout
 
 
+# Hardcore monkeypatching action.
+import jingo.monkey
+jingo.monkey.patch()
```

Comment:
I'm not sure, we'd have to test. When I did this it worked w/o the extra import but I didn't check receiptcheck.